framework/source/class/qx/event/handler/Mouse.js

```diff
@@ -375,6 +377,17 @@ qx.Class.define("qx.event.handler.Mouse",
 
       this.__fireEvent(domEvent, type, target);
 
+      /*
+       * In order to normalize middle button click events we
+       * need to fire an artificial click event if the client
+       * fires auxclick events for non primary buttons instead.
+       * 
+       * See https://github.com/qooxdoo/qooxdoo/issues/9268
+       */
+      if (type == "auxclick" && domEvent.button == 1) {
+        this.__fireEvent(domEvent, "click", target);
```

Comment:
Can we make this configurable? I never knew that middle mouse button would fire "click" and I consider it undesirable. If that's the way it worked before, it should continue to, but It would be really nice to have a property can can have it generate its own "auxclick" event *instead* of the "click" event. (It probably doesn't hurt for it to always generate the "auxclick" event; it's only "click" that I think should allow for only left-click.